docs/source/configuration.rst

```diff
@@ -147,6 +147,7 @@ default.
           keywords:  # list of related keywords
               - observations
               - monitoring
+          geojsonld: True # use default geojsonld behavior (see Linked Data section)
```

Comment:
Should be (JSON) `true`

Reply:
Thanks to @webb-ben for changing this. All checks pass now.